services/dashboard/views/apps/streamingapp/streamingapp.html

```diff
@@ -8,13 +8,13 @@
       </h4>
       <h5 ng-bind="app.description"></h5>
     </div>
-    <div class="col-md-2 col-sm-3 col-xs-4">
+    <div class="col-md-2 col-sm-3 col-xs-3">
```

Comment:
Why "Application Clock" need to align with actorPath field of another table? As they are two different tables, it looks to me not that necessary to do the align.

I tried the "col-xs-4", "col-xs-8", "col-xs-12" you suggested. Here is the screenshot after the change. If you think this looks better, I can update the patch.
![align12](https://cloud.githubusercontent.com/assets/2595532/12382254/f901205e-bdd2-11e5-8988-76fc3ea46d44.jpg)